dcos/subcommand.py

```diff
@@ -219,8 +244,32 @@ def _write_package_json(pkg):
         json.dump(package_json, package_file)
 
 
+def _get_cli_binary(resources):
+    """Find compatible cli binary, if one exists
+
+    :param resources: resource.json
+    :type resources: {}
+    :returns: cli binary
+    :rtype: str or None
+    """
+    cli = resources.get("cli")
+    if cli and cli.get("binaries"):
+        arch = platform.architecture()[0]
+        if arch != "64bit":
+            logger.debug("Unsupported archecture {}\n", arch)
+            return None
+        system = platform.system().lower()
+        if system == "darwin":
+            system = "osx"
```

Comment:
We could also just call it `darwin` in the package spec. Maybe that is the correct platform name.